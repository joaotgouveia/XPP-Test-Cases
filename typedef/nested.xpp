type my_type1 = int;
type my_type2 = my_type1;
(
let f = fn x: my_type2 => { x }; f(1)
);;

type my_int = int;
type my_struct = struct { a: my_int };
(
let f = fn a: my_struct => { a.a };
f({ a = 1 })
);;
