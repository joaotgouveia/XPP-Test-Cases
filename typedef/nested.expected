Value: 1
Type: int

Value: 1
Type: int
